src/main/groovy/org/ajoberstar/grgit/Grgit.groovy

```diff
@@ -21,6 +21,7 @@ import org.ajoberstar.grgit.util.JGitUtil
 import org.ajoberstar.grgit.util.OpSyntaxUtil
 
 import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.lib.StoredConfig
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The [org.eclipse.jgit.lib.StoredConfig] import is never referenced [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=grvy%3Aorg.codenarc.rule.imports.UnusedImportRule)